test/unit/lib/pa11y.test.js

```diff
@@ -21,7 +21,6 @@ describe('lib/pa11y', () => {
 	let puppeteer;
 	let extend;
 	let semver;
-	let promiseTimeout;
```

Comment:
🚨 Potentially silly question incoming klaxon: why don't we need a unit test on the timeout anymore?

Reply:
I'm not fully convinced one way or the other either.  Overall though, the relevant sections of this file were:

- getting rid of the actual timeout functionality, maybe so it didn't trip up the unit tests
- while making sure that its provider `p-timeout` was still called, somewhere/anywhere during its execution (it isn't checking that a timeout happens or when)

The effects of this PR's changes:

- the default timeout is still in play, but it doesn't occur during the unit tests because they're speedy
- the test of the timeout's actual effect is made by the integration test
    - ... but, as described above, this was the case before
- the fact that the real object rather than a mock is in use could be argued to build confidence in the test (I'm reaching here)

Confidence level: 75%

Reply:
75% is enough convincing for me! thanks for the explanation :D I am a big newb with this sort of thing so just needed a bit of handholding, heh.